<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="hemsireKayit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>231</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Hasta Kayıt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hemsireGor">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>231</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Hasta Kaydı Görüntüle</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
